CHANGES.md

```diff
@@ -4,7 +4,7 @@ Change Log
 ### 1.42 - 2018-02-01
 
 * Breaking changes
-  *
+  * The clock does not animate by default. Set the viewer `ClockViewModel.shouldAnimate` property to `true` to enable animation.
```

Comment:
Let's use `viewer.clock.shouldAnimate` here instead of "viewer `ClockViewModel.shouldAnimate`"